markdown/usage/installation.md

```diff
@@ -1,28 +1,15 @@
 ---
 title: Installation
-subtitle: Installing the building blocks needed for running nf-core pipelines.
+subtitle: Installing the software requirements needed for running nf-core pipelines.
 ---
 
-This documentation assumes that you have already read the [introduction](introduction) and are familiar with the tools described below.
-
-## Table of contents
-<!-- Install Atom plugin markdown-toc-auto for this ToC -->
-<!-- TOC START min:2 max:3 link:true asterisk:true -->
-* [Install nextflow](#install-nextflow)
-* [Install the pipeline](#install-the-pipeline)
-  * [Automatic](#automatic)
-  * [Offline](#offline)
-  * [Development](#development)
-* [Pipeline configuration](#pipeline-configuration)
-  * [Docker](#docker)
-  * [Singularity](#singularity)
-  * [Conda](#conda)
-  * [Configuration profiles](#configuration-profiles)
-* [Reference genomes](#reference-genomes)
-<!-- TOC END -->
-
-## Install nextflow
-Nextflow runs on most POSIX systems (Linux, Mac OSX etc). It can be installed by running the following commands:
+## Nextflow
+
+All nf-core pipelines use Nextflow - this is the only absolute requirement for running pipelines.
```

Comment:
Unless you are running in `local` mode you would need to have `Docker`, `Singularity` or `Conda` installed too.

Reply:
Yes, but you _could_ have all of that software installed, so Nextflow is the only _absolute_ requirement 😉 

Ok, this is a bit unhelpful..